RoundhousE.MSBuild dosen't recognize RecoveryMode="Simple"

I'm trying to create a database with RoundhousE.MSBuild with the recovery model "Simple" but I cannot get it to work. First
try:
 <Target Name="CreateDatabaseWithRecoveryModelSimple1">
    <!--Trying RecoveryMode="Simple" as in the Sample.xml-->
    <Roundhouse
      DatabaseName="TestRH"
      ConnectionString="Data Source=(local);Integrated Security=true;Application Name=RoundhousE;"
	    RepositoryPath="MyRepository"
	    DatabaseType="roundhouse.databases.sqlserver.SqlServerDatabase, roundhouse"
	    OutputPath="$(MSBuildProjectDirectory)\Logs\RoundhousE_runs"
	    Silent="true"
      RecoveryMode="Simple"
      CreateDatabaseCustomScript="custom_db_create.sql"
      />
  </Target>

Then I try:
  <Target Name="CreateDatabaseWithRecoveryModelSimple2">
    <!--Trying to use fully qualified-->
    <Roundhouse
      DatabaseName="TestRH"
      ConnectionString="Data Source=(local);Integrated Security=true;Application Name=RoundhousE;"
	    RepositoryPath="MyRepository"
	    DatabaseType="roundhouse.databases.sqlserver.SqlServerDatabase, roundhouse"
	    OutputPath="$(MSBuildProjectDirectory)\Logs\RoundhousE_runs"
	    Silent="true"
      RecoveryMode="roundhouse.databases.RecoveryMode.Simple"
      CreateDatabaseCustomScript="custom_db_create.sql"
      />
  </Target>


Both results in the following error:
error MSB4030: "Simple" is an invalid value for the "RecoveryMode" parameter of the "Roundhouse" task. The "RecoveryMode" parameter is of type "roundhouse.databases.RecoveryMode".

<!---
@huboard:{"milestone_order":131.0,"order":131.0}
-->

--- a/product/roundhouse.tasks/Roundhouse.cs
+++ b/product/roundhouse.tasks/Roundhouse.cs
@@ -16,6 +16,8 @@
 
     public sealed class Roundhouse : ITask, ConfigurationPropertyHolder
     {
+        private RecoveryMode recoveryMode;
+
         #region MSBuild
 
         public IBuildEngine BuildEngine { get; set; }
@@ -119,7 +121,35 @@
 
         public bool WithTransaction { get; set; }
 
-        public RecoveryMode RecoveryMode { get; set; }
+        RecoveryMode ConfigurationPropertyHolder.RecoveryMode
+        {
+            get
+            {
+                return this.recoveryMode;
+            }
+            set
+            {
+                this.recoveryMode = value;
+            }
+        }
+
+        public string RecoveryMode
+        {
+            get
+            {
+                return this.recoveryMode.ToString();
+            }
+            set
+            {
+                RecoveryMode result;
+                if (!Enum.TryParse(value, true, out result))
+                {
+                    throw new ArgumentOutOfRangeException("value", value, "The value of 'RecoveryMode' must be one of these values: 'NoChange', 'Simple' or 'Full'.");
+                }
+
+                this.recoveryMode = result;
+            }
+        }
 
         [Obsolete("Use RecoverMode=Simple now")]
         public bool RecoveryModeSimple { get; set; }

